src/queries/index.js

```diff
@@ -1,72 +1,96 @@
-import gql from 'graphql-tag'
+import { parse as gqlParse } from 'graphql'
+import { parse } from 'gotql/dist/modules/parser'
 
-export const GET_BUSINESSES = gql`
-fragment businessFields on businesses {
-  id
-  name
-  short_description
-  long_description
-  locations {
-    location
-    name
-  }
-  territories {
-    territory {
-      id
-      name
-      description_url
+const BUSINESS_FIELDS = [
+  'id', 'name', 'short_description', 'long_description',
+  {
+    locations: { fields: ['id', 'location', 'name', 'postal_address'] }
```

Comment:
I will need to see if this provides autocomplete in the editor. That would be nice